base/shell/cmd/assoc.c

```diff
@@ -178,7 +191,6 @@ AddAssociation(LPTSTR extension, LPTSTR type)
     return 0;
```

Comment:
Might as well remove the code duplication here, and always close the keys then check what return_val was.